src/screencast/pipewire_screencast.c

```diff
@@ -44,6 +86,30 @@ static struct spa_pod *build_format(struct spa_pod_builder *b, enum spa_video_fo
 	return spa_pod_builder_pop(b, &f[0]);
 }
 
+static uint32_t build_formats(struct spa_pod_builder *b, struct xdpw_screencast_instance *cast, const struct spa_pod **params) {
```

Comment:
Use `const struct spa_pod *params[static 2]` to make it clearer this expects exactly 2 params.